apollo-router/src/plugins/coprocessor/execution.rs

```diff
@@ -19,7 +18,7 @@ use crate::plugins::coprocessor::EXTERNAL_SPAN_NAME;
 use crate::services::execution;
 
 /// What information is passed to a router request/response stage
-#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize, JsonSchema)]
```

Comment:
I notice `condition` field is skip during serialization in multiple places, e.g.:
https://github.com/apollographql/router/blob/f4ff8225124318c22c7748808ee3efbc9bd7eb8f/apollo-router/src/plugins/coprocessor/mod.rs#L316-L317

That made me wonder where we use serialization of this type and why it's valid to skip conditions. I learned that we don't use serialization for those structs so I removed them.